### ISSUE TITLE
Banner: Add Play/Pause button

### ISSUE THREAD

**Issue by hannessolo** [hlxsites/maidenform#100]
see: https://github.com/hlxsites/maidenform/pull/95#issuecomment-1398493871

We would like to add a play/pause button to the banner auto play functionality. 

@karigaron17 will look into how the UI for this should be designed.

**Comment by herzog31** [comment]
Confirmed with Jordan, that we don't change the promo banners for now.
